src/libvirt-common.js

```diff
@@ -1120,13 +1120,13 @@ export function createVm({
         profile,
         useCloudInit,
     ], opts)
-            .done(() => {
+            .then(() => {
                 finishVmCreateInProgress(vmName, connectionName);
                 clearVmUiState(vmName, connectionName);
             })
-            .fail((exception, data) => {
+            .fail(ex => {
```

Comment:
This is the one place which breaks [a single test on Debian stable](https://logs.cockpit-project.org/logs/pull-279-20210728-073549-c6fdeec1-debian-stable/log.html#14-2), presumably because of some buggy cockpit.js promise API in cockpit 239. This needs at least a comment, and what fixing it depends on.